cdap-api/pom.xml

```diff
@@ -52,6 +52,10 @@
       <artifactId>tephra-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>co.cask.tephra</groupId>
```

Comment:
I need those classes to wrap the run method in a transaction.  Is there a better way to do this?